packages/migrate/test/__snapshots__/migrate.test.ts.snap

```diff
@@ -111,23 +111,6 @@ export default class WithMissingInterface extends Component<WithMissingInterface
 "
 `;
 
-exports[`fix > .gts > component signature codefix > with misssing jsdoc param name 1`] = `
-"import Component from \\"@glimmer/component\\";
-
-export class Something extends Component {
-  /** */
-  // @ts-expect-error @rehearsal TODO TS7019: Rest parameter 'args' implicitly has an 'any[]' type.
-  reset(...args) {
-    console.log(args);
-  }
-
-  shouldExist() {
-    console.log(\\"should exist in output\\");
-  }
-}
-"
-`;
-
```

Comment:
These should not be removed? Using test.only

Reply:
Can't find test with this name, seems it was renamed... there is same result in `exports[`fix > .gts > strips jsdoc param with missing name 1`] = ``.

Same with the test below.

Reply:
![image](https://github.com/rehearsal-js/rehearsal-js/assets/1502496/e8e98a9a-96c4-4a89-a727-6c96e56d6d00)